lib/mix/lib/mix/tasks/test.ex

```diff
@@ -569,6 +575,22 @@ defmodule Mix.Tasks.Test do
     end
   end
 
+  defp normalize_max_failures(opts) do
+    case Keyword.fetch(opts, :max_failures) do
+      value when value in ["infinity", ":infinity"] ->
```

Comment:
I think this should be `{:ok, value}`

Reply:
🤔And I think the `:error` value cannot be matched

Reply:
oops